src/components/SearchBar/SearchBar.test.tsx

```diff
@@ -0,0 +1,17 @@
+import SearchBar from './SearchBar';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { vitest } from 'vitest';
+import userEvent from '@testing-library/user-event';
+
+describe('SearchBar', () => {
+  const onTextChange = vitest.fn();
+
+  it('should render correct', async () => {
```

Comment:
if u describe test as render correct so show a snapshot or check that all components are rendered, but check that something was called is another story

Reply:
corrected describe )